docs/config/node.rst

```diff
@@ -658,11 +658,10 @@ Javascript language
 ===================
 
 **lang.js.enabled**
-  | *Default:*  ``false``
+  | *Default:*  ``true``
   | *Runtime:*  ``no``
 
-  Setting to enable the Javascript language. As The Javascript language is an
-  experimental feature and is not securely sandboxed its disabled by default.
+  Setting to enable or disable the :ref:`JavaScript UDF support <udf_lang_js>`.
```

Comment:
I would maybe drop 'the' in this sentence, so that it reads "Setting to enable or disable JavaScript UDF support", or to phrase it as "Setting to enable or disable the Javascript UDF language".